specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-05-30-preview/share.json

```diff
@@ -2148,9 +2130,9 @@
       "type": "string",
       "x-ms-parameter-location": "method"
     },
-    "orderby": {
+    "orderBy": {
       "in": "query",
-      "name": "orderby",
+      "name": "orderBy",
```

Comment:
`orderby` must be all lower case per the [Azure REST API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options).

Reply:
name must be `orderby`. You can have a x-ms-client-name to whatever you like e.g. `orderBy`. https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-client-name